src/client/js/script.js

```diff
@@ -62,5 +62,582 @@ $(document).ready(function() {
+{
+  menu
+    .stop(true)
+    .fadeOut(100, function() {
+    menu
+    .css(
+	{
+    'left'         : '',
+    'top'          : '',
+    'display'      : '',
+    'opacity'      : '',
+    'height'       : '',
+    'marginTop'    : '',
+    'marginBottom' : ''
+	}
+    )
+    .attr('aria-expanded', 'false');
+  });
+};
+
+$(document).ready(NavMain.init);
+
+})();
+
+/*
+nav-main.js
+*/
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(document).ready(function() {
+  if ($('#nav-main').length === 0) {
+    return;
+  }
+
+  var main_menuitems = $('#nav-main [tabindex="0"]');
+  var prev_li, new_li, focused_item;
+
+  $('#nav-main > .has-submenus > li').bind('mouseover focusin', function(event) {
+    new_li = $(this);
+    if (!prev_li || prev_li.attr('id') !== new_li.attr('id')) {
+      // Open the menu
+      new_li.addClass('hover').find('[aria-expanded="false"]').attr('aria-expanded', 'true');
+      if (prev_li) {
+        // Close the last selected menu
+        prev_li.dequeue();
+      }
+    } else {
+      prev_li.clearQueue();
+    }
+  }).each(function(menu_idx) {
+    var menuitems = $(this).find('a');
+
+    menuitems.mouseover(function(event) {
+      this.focus(); // Sometimes $(this).focus() doesn"t work
+    }).focus(function() {
+      focused_item = $(this);
+    }).each(function(item_idx) {
+      // Enable keyboard navigation
+      $(this).keydown(function(event) {
+        var target;
+        if(event.altKey || event.ctrlKey || event.metaKey || event.shiftKey) {
+          return true;
+        }
+        switch (event.keyCode) {
+          case 33: // Page Up
+          case 36: // Home
+            target = menuitems.first();
+            break;
+          case 34: // Page Down
+          case 35: // End
+            target = menuitems.last();
+            break;
+          case 38: // Up
+            target = (item_idx > 0) ? menuitems.eq(item_idx - 1)
+                                    : menuitems.last();
+          case 37: // Left
+            break;
+          case 40: // Down
+            target = (item_idx < menuitems.length - 1) ? menuitems.eq(item_idx + 1)
+                                                       : menuitems.first();
+            break;
+            target = (menu_idx > 0) ? main_menuitems.eq(menu_idx - 1)
+                                    : main_menuitems.last();
+            break;
+          case 39: // Right
+            target = (menu_idx < main_menuitems.length - 1) ? main_menuitems.eq(menu_idx + 1)
+                                                            : main_menuitems.first();
+            break;
+        }
+        if (target) {
+          target.get(0).focus(); // Sometimes target.focus() doesn't work
+          return false;
+        }
+        return true;
+      });
+    });
+  });
+
```

Comment:
@Slayslot It is throwing error here. Please check your code if its working or not because it isn't.